app/v2/notifications/post_notifications.py

```diff
@@ -49,6 +50,7 @@
 
 @v2_notification_blueprint.route('/<notification_type>', methods=['POST'])
 def post_notification(notification_type):  # noqa: C901
+    created_at = datetime.now(timezone.utc)
```

Comment:
same as above wrt to `created_at`

Reply:
This one is a little different from the others, and part of the AC. We want to track total time from the time we receive an authenticated request until just before we have sent it to the provider. This value is passed to notification creation and is saved in the DB as the created_at timestamp.